resources/aws/subnet.go

```diff
@@ -198,3 +198,46 @@ func (s *Subnet) AddHostVPCRoute(routeTable *RouteTable, vpcPeeringConnection *V
 
 	return nil
 }
+
+func (s *Subnet) AddGuestVPCRoute(routeTable *RouteTable, vpcPeeringConnection *VPCPeeringConnection) error {
```

Comment:
Maybe having a private method in `Subnet` with the common logic might help, with a flag for creating or deleting, I think the code would remain very understandable, but not sure about the conventions in this sense.

Reply:
I changed this around quite a bit, not sure if I made it better or worse but having this in `resources/aws/subnet.go` did not feel right.